mne/io/nihon/nihon.py

```diff
@@ -272,10 +274,19 @@ def _read_nihon_annotations(fname):
             n_logs = np.fromfile(fid, np.uint8, 1)[0]
             fid.seek(t_blk_address + 0x14)
             t_logs = np.fromfile(fid, '|S45', n_logs)
-            for t_log in t_logs:
+            if version == _valid_headers[-1]:
+                fid.seek(0x92 + (t_block + 22) * 20)
+                t_sub_blk_address = np.fromfile(fid,np.uint32,1)[0]
```

Comment:
```suggestion
                t_sub_blk_address = np.fromfile(fid, np.uint32, 1)[0]
```

Style error. please run `make flake` from the root folder of the MNE clone to catch all the other ones, and fix them.